### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by manuelschroederdev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,19 +15,19 @@
...
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.2.2",
-    "@commitlint/config-conventional": "^19.2.2",
-    "@typescript-eslint/parser": "^7.16.1",
-    "eslint": "^8.57.0",
+    "@commitlint/cli": "^19.5.0",
+    "@commitlint/config-conventional": "^19.5.0",
+    "@typescript-eslint/parser": "^8.12.1",
+    "eslint": "^9.13.0",
```
Comment (alvarosabu) [inline package.json:24]: @manuelschroederdev we shouldn't update to v9 of eslint until we get the new pnpm setup with eslint flat config and `@storyblok/eslint-config` package

File: package.json [hunk 2]
```diff
@@ -15,19 +15,19 @@
...
+    "prepare": "husky"
   },
   "devDependencies": {
-    "@commitlint/cli": "^19.2.2",
-    "@commitlint/config-conventional": "^19.2.2",
-    "@typescript-eslint/parser": "^7.16.1",
-    "eslint": "^8.57.0",
+    "@commitlint/cli": "^19.5.0",
+    "@commitlint/config-conventional": "^19.5.0",
+    "@typescript-eslint/parser": "^8.12.1",
```
Comment (alvarosabu) [inline package.json:23]: Probably would avoid updating this major as well for now

**Review by alvarosabu — CHANGES_REQUESTED** [review]
@manuelschroederdev I would just update minors and patches since majors have some breaking changes

**Comment by manuelschroederdev** [comment]
Hey @alvarosabu, thanks for checking, I've made the changes.

**Review by alvarosabu — APPROVED** [review]
Thanks @manuelschroederdev

**Pull request merged**